tests/integration/components/new-project-modal/component-test.ts

```diff
@@ -1,6 +1,7 @@
 import { A } from '@ember/array';
 import Service from '@ember/service';
 import { render } from '@ember/test-helpers';
+import { setupIntl } from 'ember-intl/test-support';
```

Comment:
This file is an example of what needs to happen for component integration tests to wok